dev.js

```diff
@@ -1,5 +1,6 @@
 require('shelljs/global');
 var fs = require('fs');
+var isWin = ('win32' === require('os').platform());
```

Comment:
This shouldn't be necessary, the replacement can be done regardless of whether it's Windows or not